nix/haskell.nix

```diff
@@ -110,10 +110,6 @@ haskell-nix: haskell-nix.cabalProject' [
         filter = haskell-nix.haskellSourceFilter;
       };
 
-      # materialized = ./materialized/plan-nix;
-
-      sha256map = import ./sha256map.nix;
```

Comment:
Seems like the sha256map was unused anyway 🤷‍♂️ (unless there is some clever fallback)

Reply:
No, removing this on `master` results in:

```
direnv: loading ~/IOHK/cardano-wallet/.envrc
direnv: using flake
warning: Git tree '/Users/anviking/IOHK/cardano-wallet' is dirty
[1/0/1 built, 0.0 MiB DL] building cardano-wallet-stack-to-nix-pkgsdirenv: ([/usr/local/bin/direnv export fish]) is taking a while to execute. Use CTRL-C to give up.
error: Cannot find Git revision '7497a29cb998721a9068d5725d49461f2bba0e7a' in ref 'master' of repository 'https://github.com/input-output-hk/optparse-applicative'! Please make sure that the rev exists on the ref you've specified or add allRefs = true; to fetchGit.
(use '--show-trace' to show detailed location information)
warning: Git tree '/Users/anviking/IOHK/cardano-wallet' is dirty
direnv: renewed cache
direnv: export +XDG_DATA_DIRS
```
when calling `direnv reload`. But with the switch to `cabal` in #3293 (merged to #3315 ) we this is redundant, because haskell.nix reads the `--sha256` annotations in cabal.project.